package.json

```diff
@@ -88,6 +88,9 @@
     "transformIgnorePatterns": [
       "node_modules/(?!(jest-)?react-native|react-navigation)"
     ],
+    "moduleNameMapper": {
+      "\\.(png|gif|jpeg|jpg)$": "RelativeImageStub"
+    },
```

Comment:
I ran this on Windows and it couldn't deal with imported images during unit tests. This seems to be the recommended fix.